spec/section_spec.rb

```diff
@@ -85,5 +86,26 @@ class Page < SitePrism::Page
       expect(section).to respond_to :execute_script
       expect(section).to respond_to :evaluate_script
     end
+
+    it 'responds to Capybara::Node::Element methods' do
+      expect(section).to respond_to :visible?
+      expect(section).to respond_to :text
+    end
+  end
+
+  describe 'page' do
+    subject(:section) { SitePrism::Section.new('parent', root_element).page }
+
+    let(:root_element) { 'root' }
+
+    it { is_expected.to eq 'root' }
```

Comment:
Can we wrap both equality matchers in `()` (Again aware of differences, will be a tidy up PR from myself)